corona-warn-app/cwa-app-android
# DCC Ticketing: Consent I navigation (EXPOSUREAPP-10644)

## Whats new
Navigation to from UQS to Dcc Ticketing Consent I screen

## Jira
[Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10644) 
[Story](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10194)

## How to test

1. Use cwa cli command: `cwa ti gen -o`
2. Scan QR Code
3. Check navigation to Consent I screen
4. Check that fields provider and subject are set
